servant-swagger/servant-swagger.cabal

```diff
@@ -1,5 +1,5 @@
 name:                servant-swagger
-version:             1.1.11
+version:             2.2
```

Comment:
You've changed `version` (which is the library version) instead of `cabal-version` (which is the format used by the Cabal file.

Reply:
damn sorry. Fixed.